### PR TITLE
Feature front articles template

### PR DESCRIPTION
**Pull request by BlakeLatch**
Finished article template, linked to news page, and updated app.py with the route for this page. Rebased before pull request.
image

### CODE REVIEW

**Comment by BlakeLatch** [comment]
Please don't merge request until I reply to this msg. I will update this once I merge Aryan's branch with the Develop one.

**Comment by BlakeLatch** [comment]
Ready to merge

**Comment by BlakeLatch** [comment]
Updated RE discussion with Shane + added other changes I was working on to avoid conflict issues with multiple Pull Requests.

**Review by SDzartov — APPROVED** [review]

**Pull request merged**
